lib/src/main/java/org/exoplatform/social/addons/updater/IdentityMigrationService.java

```diff
@@ -276,19 +277,25 @@ public void doRemove() throws Exception {
     } finally {
       MigrationContext.setIdentitiesCleanupFailed(identitiesCleanupFailed);
       if (identitiesCleanupFailed.size() > 0) {
-        LOG.warn("Cleanup failed for " + identitiesCleanupFailed.size() + " identities");
+        LOG.warn("Cleanup failed for {} identities", identitiesCleanupFailed.size());
       }
-      LOG.info(String.format("| / END::cleanup Identities migration for (%s) identity consumed %s(ms)", offset, System.currentTimeMillis() - t));
+      LOG.info("| / END::cleanup ({}) identities duration_ms={}", offset, System.currentTimeMillis() - t);
     }
   }
 
   private Identity migrateIdentity(Node node, String jcrId) throws Exception {
     String providerId = node.getProperty("soc:providerId").getString();
     String remoteId = node.getProperty("soc:remoteId").getString();
+    String name = node.getName();
+
+    // Ensure the nodes are coherents
+    if (!name.equals("soc:" + remoteId)) {
+      throw new Exception(String.format("Identity has different name and remote_id provider_id=%s name=%s remote_id=%s", providerId, remoteId, name));
```

Comment:
We throw exception mean this node will be marked as failed in migration? So, its activities won't be migrated also?

I'm afraid this line will make the migration stop as we are checking if there is any identity was not migrated, we will stop migration.